book/08-customizing-git/sections/attributes.asc

```diff
@@ -329,9 +329,9 @@ The substitutions can include for example the commit message and any `git notes`
 [source,console]
 ----
 $ echo '$Format:Last commit: %h by %aN at %cd%n%+w(76,6,9)%B$' > LAST_COMMIT
-$ git commit -am 'export-subst uses git log's custom formatter
+$ git commit -am 'export-subst uses git log'\''s custom formatter
```

Comment:
No, it does not work. You can try with a simple echo :
$ echo 'export-subst uses git log\'s custom formatter'
> ^C
$ echo 'export-subst uses git log'\''s custom formatter'
export-subst uses git log's custom formatter

With the first command, bash is waiting for rest of the command, which shows that using the back slash in the string does not do what you wanted.
With the second command, you get the expected result.